src/pages/chat.tsx

```diff
@@ -0,0 +1,356 @@
+                                            Thank you so much!
+                                        </span>
+                                    </div>
+                                    <div>
+                                        <span className="px-4 py-2 rounded-lg inline-block rounded-bl-none bg-gray-300 text-gray-600">
+                                            I am going to try doing that, you
+                                            have a good day.
+                                        </span>
+                                    </div>
+                                </div>
+                                <div className="w-6 h-6 rounded-full order-1 flex items-center">
+                                    <Image
+                                        src={avatarFemale}
+                                        alt="Female Avatar"
+                                        className="avatar"
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                        <div className="chat-message">
+                            <div className="flex items-end justify-end">
+                                <div className="flex flex-col space-y-2 text-xs max-w-xs mx-2 order-1 items-end">
+                                    <div>
+                                        <span className="px-4 py-2 rounded-lg inline-block rounded-br-none bg-dominant text-white ">
+                                            You too!
+                                        </span>
+                                    </div>
+                                </div>
+                                <div className="w-6 h-6 rounded-full order-1 flex items-center">
+                                    <Image
+                                        src={avatarMale}
+                                        alt="Male Avatar"
+                                        className="avatar"
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    {/* Messages Window End */}
+
+                    <div className="border-t-2 border-gray-200 px-4 pt-4 mb-2 sm:mb-0 pb-3">
+                        <div className="relative flex">
+                            <input
+                                type="text"
+                                placeholder="Write your message!"
+                                className="w-full focus:outline-none focus:placeholder-gray-400 text-gray-600 placeholder-gray-600 pl-6 bg-gray-200 rounded-md py-3"
+                            />
+                            <div className="absolute right-0 items-center inset-y-0 flex">
+                                {/* Smiley */}
+                                <button
```

Comment:
Do we really need this emoji button? Software Engineering rule is "If you don't need it now, don't add it."

Reply:
We don't really need it right now and it can be easily added later, so I will be removing it in the new commit I make.